README.md

```diff
@@ -104,7 +104,13 @@ bundle exec rake test:extensions
 bundle exec rake test:all
 ```
 
-It is recommended to run the last command at least once right before pushing.
+### Run absolutely all the tests plus rubocop and bundle audit
+
+```bash
+bundle exec rake
+```
+
+It is recommended to run the last command at least once right before pushing. Also, running this command assures you that what you get locally is the same that you will get in Travis.
```

Comment:
This paragraph is a little odd. Travis is introduced with no prior explanation, or with any indication of why it's important, and with the implication that getting something from Travis is a goal. But this all seems slightly back-to-front. Travis merely checks that all these things pass — **it** is not really the goal by itself. 

If we're going to spell this all out, then I think we should do it even more explicitly, explaining what checks we run{{1}}, and explaining that we have Travis set up to run those checks automatically, and that _you_, dear contributor, can run exactly the same checks through a simple `bundle exec rake`.

---

{{1}} ideally in a semi-standardised way that we can drop in to multiple projects